### ISSUE TITLE
Charge extractor of interleaved pedestals should be FixedWindowSum

### ISSUE THREAD

**Issue by FrancaCassol** [cta-observatory/cta-lstchain#801]
In order to proceed to a not biased estimation interleaved pedestals (and to recalculate the gains with DL1 images) the charge extractor of pedestals should be FixedWindowSum, not local LocalPeakWindowSum

**Comment by maxnoe** [comment]
But we don't want 0 bias, we want *the same* bias as for the physics events, correct?

For that, we have to use the same extractor as we use for the physics events.

**Comment by moralejo** [comment]
No, it is like for muons, we do indeed want unbiased extractor (e.g. if you want to re-do the F-factor method with interleaved events)

**Comment by maxnoe** [comment]
But this will make the pedestal unusable as background estimate for the cleaning, correct?

**Comment by moralejo** [comment]
Yes, that is also right, for the use we make of it now (clean stars)  we want to have the biased extractor

**Comment by moralejo** [comment]
Let's however not start working on, or even discussing this, before the new release is out.

**Comment by FrancaCassol** [comment]
Ok, let me go ahead with the level B calibration, which should help to tag stars and hopefully to improve the cleaning method. For the moment we do not touch this

**Comment by moralejo** [comment]
So, to clarify: the point is whether to *keep* calibrated images of interleaved pedestals obtained with *fixed window* extractor, for the purpose of facilitating the level B calibration, right?

**Comment by FrancaCassol** [comment]
Yes, this is the point

**Comment by FrancaCassol** [comment]
Hi @maxnoe ,

> But this will make the pedestal unusable as background estimate for the cleaning, correct?

going back to this, why do you think that?

**Comment by FrancaCassol** [comment]
Actually this issue is due to the fact that we are going from R0 directly to DL1.
In CTA, the steps will be from R1 to DL0 in ACADA and from DL0 to DL3 in DPPS.

DL0 data will contain waveforms (in the case of pedestal and FF events no volume reduction must be applied hopefully), so calibration DL) data data can be easily reprocessed with the proper charge integrator for a level B gain estimation.  

But, thinking to the level B calibration for LST1 now, we have two options:

1) we process pedestal and FF events as for the level A calibration (12 ns windows and fixed window for pedestals), so one can directly use  DL1 images for the level B calibration

2) we start again from R0 data (with the help of events indexes from DL1 data)

Option 1 would be easier, do you see contraindications for it?

**Comment by maxnoe** [comment]
> But this will make the pedestal unusable as background estimate for the cleaning, correct?

Because when not using the same extractor settings, the values estimated from pedestal events will not be the same as the expected background in the physics events.

It might not matter too much for cleaning, you can just adapt the factor probably. But for things like likelihood fits to the light distribution, you need an estimate of the background for the extractor you are using. Not a different one.

**Comment by FrancaCassol** [comment]
> Because when not using the same extractor settings, the values estimated from pedestal events will not be the same as the expected background in the physics events.

Which values do you mean? What you want is to estimate with pedestal events is the pedestal mean and std (this last permitting to estimate NSB and stars). If you use the local peak integrator you bias the mean pedestal estimation to higher values which depend on the NSB (i.e. pedestal std) and you overestimate the mean pedestal baseline. 

> 
> It might not matter too much for cleaning, as you can just adapt the factor probably. But for things like likelihood fits to the light distribution, you need an estimate of the background for the extractor you are using. Not a different one.

This is not a standard reconstruction, so I do not know how they proceed, but  I guess there are enough background data in each image to permit an estimation of the local peak NSB level without using pedestal events (are they really used them?)
